frontend/src/routes.js

```diff
@@ -45,13 +45,13 @@ export default (
     <Route path="/:slug/connect/:provider" component={ConnectProvider} />
     <Route path="/:slug/edit-twitter" component={EditTwitter} />
     <Route path="/:slug" component={PublicPage} />
-    <Route path="/:slug/donate/:amount" component={DonatePage} />
-    <Route path="/:slug/donate/:amount/:interval" component={DonatePage} />
+    <Route path="/:slug/:verb(donate|pay|contribute)/:amount/:interval/:description" component={DonatePage} />
```

Comment:
It would be good to add a comment to let others know that these verbs don't work.

Or maybe better to remove them from code and add them in the comment only.